mail-filter/imapfilter/imapfilter-2.7.6.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} luajit )
+
+inherit lua-single toolchain-funcs
+
+DESCRIPTION="An IMAP mail filtering utility"
+HOMEPAGE="https://github.com/lefcha/imapfilter"
+SRC_URI="https://github.com/lefcha/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+
+RDEPEND="
+	dev-libs/openssl:0=
+	dev-libs/libpcre2
+	${LUA_DEPS}"
+DEPEND="${RDEPEND}"
+
+DOCS="AUTHORS NEWS README samples/*"
```

Comment:
Array please.

Reply:
`DOCS=( .. )` does not work for some reason, though that's what other ebuilds do - no docs are installed into image/usr/share/doc. The quoted string as it is above works.

Reply:
We'll poke at it next time?

Reply:
Never mind, I just figured it out. EAPI-8 DOCS=(..) apparently doesn't like '*'.